test/colony-network-mining.js

```diff
@@ -2444,15 +2444,29 @@ contract("ColonyNetworkMining", accounts => {
         // These have to be done sequentially because this function uses the total number of tasks as a proxy for getting the
         // right taskId, so if they're all created at once it messes up.
       }
+      await fundColonyWithTokens(metaColony, clny, INITIAL_FUNDING.muln(30));
 
-      // We need to complete the current reputation cycle so that all the required log entries are present
       await advanceMiningCycleNoContest({ colonyNetwork, test: this });
 
+      for (let i = 0; i < 10; i += 1) {
```

Comment:
Unless the explicit task parameters are being used (doesn't look like they are?), you could replace the following line with:

```
await setupFinalizedTask({ colonyNetwork, colony: metaColony });
```

Reply:
By chance, this task creation is actually not needed at all.